--- a/build.zig
+++ b/build.zig
@@ -4,6 +4,9 @@ fn linkFrameworks(exe: *std.Build.Step.Compile) void {
     exe.linkFramework("Cocoa");
     exe.linkFramework("Carbon");
     exe.linkFramework("CoreServices");
+    exe.root_module.addFrameworkPath(.{ .cwd_relative = "@darwin-frameworks@" });
+    exe.root_module.addSystemIncludePath(.{ .cwd_relative = "@darwin-include@" });
+    exe.root_module.addLibraryPath(.{ .cwd_relative = "@darwin-lib@" });
 }

 fn addVersionImport(b: *std.Build, exe: *std.Build.Step.Compile) void {
